test/GithubApi.Put.test.js

```diff
@@ -0,0 +1,19 @@
+const { expect } = require('chai');
+const axios = require('axios');
+
+const object = axios.create({
+  headers: {
+    Authorization: `token ${process.env.ACCESS_TOKEN}`
+  }
+});
+describe('idempotence', () => {
+  it('PUT ', async () => {
+    const response = await object.put('https://api.github.com/user/following/aperdomob');
+    expect(response.status).to.equal(204);
+    expect(response.data).to.equal('');
+  });
+  it('Following test', async () => {
+    const response = await object.get('https://api.github.com/user/following');
+    expect(response.data.find((user) => user.login === 'aperdomob')).to.not.eql(undefined);
+  });
```

Comment:
with the existing tests, you are covering points 2 and 3, for the step 4, you need to add another `it` to call again the put and then verify that you continue following the user `aperdomob`